src/sidebar/services/groups.js

```diff
@@ -107,6 +111,7 @@ export default function groups(
       if (
         directLinkedGroup &&
         !directLinkedGroup.isScopedToUri &&
+        directLinkedGroup.scopes &&
```

Comment:
It shouldn't technically be possible for `isScopedToUri` to be `true` and `scopes` to be missing, but this check makes it obvious to TS and the reader that we are accounting for this.